### PR TITLE
Added redshift stats to the library

### PR DESCRIPTION
**Pull request by dpricha89**
This enables the cloudwatch-to-graphite library to be configured for redshift stats. This adds the redshift list clusters function and a way of adding multiple dimensions to the formatter.

### CODE REVIEW

File: plumbum.py [hunk 1]
```diff
@@ -227,7 +237,8 @@ def list_dynamodb(region, filter_by_kwargs):
     'sqs': list_sqs,
     'kinesisapp': list_kinesis_applications,
     'dynamodb': list_dynamodb,
-    'billing': list_billing
+    'billing': list_billing,
+    'redshift': list_redshift
```
Comment (paul-krohn) [inline plumbum.py]: nit: trailing comma

**Review by paul-krohn — APPROVED** [review]
👍

**Review by dcrampton-sfdc — DISMISSED** [review]
Please add a description to the pull request.

**Comment by dpricha89** [comment]
@dcrampton added a description. Sorry about that.

**Review by paul-krohn — APPROVED** [review]
👍

**Review by kruxErinWillingham — APPROVED** [review]

**Pull request merged**